File: Golang/1-50/13.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	input, _ := reader.ReadString('\n')
	inputs := strings.Fields(input)
	n, _ := strconv.Atoi(inputs[0])
	k, _ := strconv.Atoi(inputs[1])

	if n == 0 {
		fmt.Println("1")
	} else {
		fmt.Println(k + 1)
	}
}

File: Dart/1-50/13.dart
import 'dart:io';

void main() {
  List<String> inputs = stdin.readLineSync()!.split(' ');
  int n = int.parse(inputs[0]);
  int k = int.parse(inputs[1]);

  if (n == 0) {
    print("1");
  } else {
    print(k + 1);
  }
}
